environment.yml

```diff
@@ -33,7 +33,8 @@ dependencies:
       - jupyter-client==6.1.5
       - lightning-bolts==0.3.4
       - matplotlib==3.3.0
-      - mlflow==1.17.0
+      - mlflow==1.17.
+      - monai==0.6.0
```

Comment:
there were a few more packages required in the dev_histo environment shouldn't they be added here?
      - azure-storage-blob==12.6.0
      - umap-learn==0.5.2
It's OK to leave out the openslide-python dependency because we're dropping it anyway in the repo